whelktool/scripts/syslib/2024/remove-kb-ebook-uri.groovy

```diff
@@ -0,0 +1,32 @@
+
+File bibIDs = new File(scriptDir, "fulltext-kb.txt")
+
+selectByIds( bibIDs.readLines() ) { bib ->
+
+	def instance = bib.graph[1]
+        
+        if (instance.associatedMedia) {
+           		List aM = instance.associatedMedia
+            	aM.removeAll { a ->
+                	a.uri instanceof String && a.uri.contains('fulltext.kb.se') ||
```

Comment:
This stuff always makes me nervous and I had to go double check the operator precedence rules. But you did it right, && binds harder than ||. For the sake of readability, i would have preferred explicit parentheses here! :)